apps/web/src/pages/api/configs/farms/[chain].ts

```diff
@@ -18,34 +14,22 @@ const handler: NextApiHandler = async (req, res) => {
     return res.status(400).json({ error: parsedChain.error })
   }
 
-  const chainId = isChainInt ? Number(parsedChain.data) : Number(chainNameToChainId[parsedChain.data])
+  const chainId = isChainInt ? Number(parsedChain.data) : Number(req.query.chain)
 
   if (!chainId) {
     return res.status(400).json({ error: 'Invalid chain' })
   }
 
   try {
-    let farms: Array<
-      SerializedFarmConfig & {
-        chainId: ChainId
-        version: 2 | 3
-      }
-    > = []
-
-    const v2FarmConfig = (await getFarmConfig(chainId)) ?? []
-    farms = farms.concat(v2FarmConfig.map((farm) => ({ ...farm, chainId, version: 2 })))
-    const v3FarmConfig = farmsV3ConfigChainMap[chainId as keyof typeof farmsV3ConfigChainMap] ?? []
-    farms = farms.concat(v3FarmConfig.map((farm) => ({ ...farm, chainId, version: 3 })))
-
-    // cache for long time, it should revalidate on every deployment
-    res.setHeader('Cache-Control', `max-age=10800, s-maxage=31536000`)
+    res.setHeader('Cache-Control', 's-maxage=60, max-age=30, stale-while-revalidate=300')
```

Comment:
Shall we set longer cache time? This is basically static configuration right? 
We can set even longer for shared cache since it will revalidate whenever redeployed